android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/DateUtil.java

```diff
@@ -117,4 +115,8 @@ public static String getDuration(long timeDiff, Locale locale, Context context)
         return duration;
 
     }
+
+    public static boolean isLeapYear(int year) {
+        return ((year % 4 == 0) && (year % 100 != 0)) || year % 400 == 0;
```

Comment:
We can use the helper method`com.vijay.jsonwizard.utils.NumericDatePickerHelper#isLeapYear` instead